include/grpc/event_engine/slice_buffer.h

```diff
@@ -88,6 +94,17 @@ class SliceBuffer {
     grpc_slice_buffer_move_first_into_buffer(&slice_buffer_, n, dst);
   }
 
+  /// Removes/deletes the last n bytes in the SliceBuffer and add it to the
+  /// other SliceBuffer
+  void RemoveLastNBytesIntoSliceBuffer(size_t n, SliceBuffer& other) {
```

Comment:
nit: would rename this to `MoveLastNBytesIntoSliceBuffer` to match the next method, which does the same thing on the first N bytes.

Reply:
Renamed.